src/Java.Interop/Java.Interop/JavaPeerableExtensions.cs

```diff
@@ -11,5 +12,40 @@ public static class JavaPeerableExtensions {
 			JniPeerMembers.AssertSelf (self);
 			return JniEnvironment.Types.GetJniTypeNameFromInstance (self.PeerReference);
 		}
+
+		public static bool TryJavaCast<
+				[DynamicallyAccessedMembers (JavaObject.ConstructorsAndInterfaces)]
```

Comment:
The existing `JavaCast<T>` only uses `Constructors`, should `Interfaces` be added as well?

Reply:
I'll leave that to @jonathanpeppers. 😅

Reply:
I added enough attributes to silence the analyzer's warnings on the original `JavaCast<T>()`.

I would recommend just putting `Constructors` for now, if no warnings appear.